### PR TITLE
tiflash: add some config item for PageStorage V3

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/10844
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by flowbehappy — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -204,7 +204,10 @@ delta_index_cache_size = 0
     ## Compression level of the TiFlash storage engine. The default value is 1. It is recommended that you set this value to 1 if dt_compression_method is LZ4, -1 (smaller compression rate, but better read performance) or 1 if dt_compression_method is zstd, and 9 if dt_compression_method is LZ4HC.
     dt_compression_level = 1
 
-    ## New in v6.2.0. Use the thread pool to handle read requests from the storage engine. The default value is false. 
+    ## New in v6.2.0. This configuration item specifies the minimum ratio of valid data in a PageStorage file. When the ratio of valid data in a PageStorage file is less than the value of this configuration, GC is triggered to compact data. The default value is 0.5.
```
Comment (lidezhu) [inline tiflash/tiflash-configuration.md:207]:
```suggestion
    ## New in v6.2.0. This item specifies the minimum ratio of valid data in a PageStorage data file. When the ratio of valid data in a PageStorage data file is less than the value of this configuration, GC is triggered to compact data in the file. The default value is 0.5.
```

Comment (lidezhu) [reply tiflash/tiflash-configuration.md:207]: Seems we always use "This item" for other config.

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -204,7 +204,10 @@ delta_index_cache_size = 0
...
+    dt_page_gc_threshold = 0.5
+
+    ## New in v6.2.0. Use the thread pool to handle read requests from the storage engine. The default value is false.
```
Comment (lidezhu) [inline tiflash/tiflash-configuration.md:210]:
```suggestion
    ## New in v6.2.0. Use the thread pool to handle read requests to the storage engine. The default value is false.
```

**Review by lidezhu — APPROVED** [review]

**Pull request merged**